packages/zoe/src/zoeService/internal-types.js

```diff
@@ -27,3 +27,80 @@
  * @param {PaymentPKeywordRecord} payments
  * @returns {Promise<Allocation>}
  */
+
+/**
+ * @callback InitInstanceAdmin
+ * @param {Instance} instance
+ * @param {InstanceAdmin} InstanceAdmin
+ * @returns {void}
+ */
+
+/**
+ * @callback GetInstanceAdmin
+ * @param {Instance} instance
+ * @returns {InstanceAdmin}
+ */
+
+/**
+ * @callback DeleteInstanceAdmin
+ * @param {Instance} instance
+ * @returns {void}
+ */
+
+/**
+ * @callback UnwrapInstallation
+ *
+ * Assert the installation is known, and return the bundle and
+ * installation
+ *
+ * @param {ERef<Installation>} installationP
+ * @returns {Promise<{ bundle: SourceBundle, installation:
+ * Installation }>}
```

Comment:
Within doc-comments, we need to do our own prettifying.
```suggestion
 * @returns {Promise<{
 *   bundle: SourceBundle,
 *   installation:Installation
 * }>}
```